companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -2114,6 +2114,40 @@ class ModuleUnionField: public UnionField<unsigned int> {
       unsigned int version;
   };
 
+  class Afhds3Field: public UnionField::TransformedMember {
+    public:
+      Afhds3Field(DataField * parent, ModuleData& module):
+        UnionField::TransformedMember(parent, internalField),
+        internalField(this, "AFHDS3")
+      {
+        ModuleData::Afhds3& afhds3 = module.afhds3;
+        internalField.Append(new UnsignedField<3>(this, minBindPower));
+        internalField.Append(new UnsignedField<3>(this, afhds3.rfPower));
+        internalField.Append(new UnsignedField<1>(this, emissionFCC));
+        internalField.Append(new BoolField<1>(this, operationModeUnicast));
+        internalField.Append(new BoolField<1>(this, operationModeUnicast));
+        internalField.Append(new UnsignedField<16>(this, defaultFailSafeTimout));
+        internalField.Append(new UnsignedField<16>(this, afhds3.rxFreq));
+      }
+
+      bool select(const unsigned int& attr) const override {
+        return attr == PULSES_AFHDS3;
+      }
+
+      void beforeExport() override {}
```

Comment:
needed?

Reply:
Needed because pure virtual function will not compile on Travis build.